src/country.c

```diff
@@ -866,3 +866,37 @@ Datum country_common_name(PG_FUNCTION_ARGS)
     country c = PG_GETARG_UINT8(0);
     PG_RETURN_TEXT_P(country_to_common_name(c));
 }
+
+PG_FUNCTION_INFO_V1(int4_to_country);
+Datum int4_to_country(PG_FUNCTION_ARGS)
+{
+    country c = PG_GETARG_INT32(0);
+
+    /* XXX we do not check input validity for performance reasons */
```

Comment:
I don't think this is a good idea as it risk that invalid values get stored on disk which makes reading back impossible